skills_ml/algorithms/occupation_classifiers/test.py

```diff
@@ -0,0 +1,45 @@
+from skills_ml.algorithms.preprocessing import IterablePipeline
+from skills_ml.algorithms.occupation_classifiers.classifiers import CombinedClassifier
+from skills_ml.algorithms.string_cleaners import nlp
+from skills_ml.job_postings.common_schema import JobPostingGeneratorType
+
+from typing import List, Callable
+from functools import partial
+from itertools import tee
+
+class OccupationClassifierTester(object):
+    def __init__(self,
+            test_data_generator: JobPostingGeneratorType,
+            preprocessing: List[Callable],
+            classifier: CombinedClassifier):
+        self.test_data_generator = test_data_generator
+        self.classifier = classifier
+        self.target_variable = self.classifier.target_variable
+        self._preprocessing_X= preprocessing
+        self._preprocessing_y = [self.target_variable.transformer]
+        self.counter = 0
+        self.pipeline_X = None
+        self.pipeline_y = None
+        self._build_pipeline()
```

Comment:
Similarly, I think this could do well as a property, or two properties (`pipeline_X` and `pipeline_y`)